nautobot/extras/tests/test_api.py

```diff
@@ -1383,7 +1387,11 @@ def test_run_job_object_var_lookup(self, mock_get_worker_count):
 
         job_result = JobResult.objects.last()
         self.assertIn("data", job_result.job_kwargs)
-        self.assertEqual(job_result.job_kwargs["data"], job_data)
```

Comment:
This was actually broken (a bug) before. Trying to rerun a job executed via the API in the UI throws a deserialization error with the JobForm.